package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "NODE_OPTIONS='-r next-logger' next start",
+    "start": "NODE_OPTIONS='-r next-logger' node next-start-cluster.js",
+    "start:legacy": "NODE_OPTIONS='-r next-logger' next start",
```

Comment:
Is it needed? Maybe `start:no-cluster`?
`legacy` is not informative

Reply:
We should also revert this, since we are removing cluster mode

Reply:
done